Examples/ByFeatures/DependencyInjection/Dependency.feature

```diff
@@ -0,0 +1,13 @@
+﻿Feature: Feeding dog
```

Comment:
Am liking this scenario, it strikes a good balance between insane-artificial and showing some of the power.